src/routes/recovery/recovery.controller.spec.ts

```diff
@@ -122,15 +129,6 @@ describe('Recovery (Unit)', () => {
         ) {
           return Promise.resolve({ status: 200, data: safe });
         }
-        if (
-          url ===
-          `${chain.transactionService}/api/v1/modules/${addRecoveryModuleDto.moduleAddress}/safes/`
-        ) {
-          return Promise.resolve({
-            status: 200,
-            data: { safes: [safe.address] },
-          });
-        }
```

Comment:
Why is this not needed anymore? I'm checking the deleted file `enable-recovery-alerts.guard.ts` and it seems a precondition for enabling and disabling the alerts was to check the module was already activated for the Safe (maybe I'm misinterpreting things), is this no longer true?

Reply:
Good point, I overlooked adding the check. I've included it (and added test coverage) as of 5b05a42855a3d6885d62caf5bd8f9020d76735b2.